lib/utils/compose.coffee

```diff
@@ -0,0 +1,746 @@
+Promise = require('bluebird')
+path = require('path')
+
+exports.appendProjectOptions = appendProjectOptions = (opts) ->
+	opts.concat [
+		{
+			signature: 'projectName'
```

Comment:
This has always been `source` or `context` in the past, context would be my preference, but project feels weird.

Reply:
I don't understand? The flag is named `--projectName` and I don't see how it relates to source, context or project.

Reply:
My mistake, I interpreted this incorrectly - LGTM.